cypress/commons/constants/generic/TestConstants.js

```diff
@@ -166,10 +174,12 @@ export const API_REGEX = {
   DATASET_DEFAULT_SECURITY: new RegExp('^' + API_ENDPOINT.DATASET_DEFAULT_SECURITY + '$'),
   DATASET_LINK: new RegExp('^' + API_ENDPOINT.DATASET_LINK + '$'),
   DATASET_SECURITY_ACL: new RegExp('^' + API_ENDPOINT.DATASET_SECURITY_ACL + '$'),
-  DATASET_SECURITY_USER_ACCESS: new RegExp('^' + API_ENDPOINT.DATASET_SECURITY_USER_ACCESS + '$'),
+  DATASET_SECURITY_USER_ACCESS: new RegExp('^' + API_ENDPOINT.DATASET_SECURITY_ACL + '$'),
```

Comment:
We already use this `DATASET_SECURITY_ACL` value in the line just above. If some functions need to use the ACL endpoints instead of "user access",  could you modify these functions instead?